c7n/resources/ebs.py

```diff
@@ -1352,8 +1353,16 @@ def process(self, volumes):
             retry(self.process_volume, client=client, volume=vol_id, tags=tags)
 
     def process_volume(self, client, volume, tags):
+        description = self.data.get('description')
+        if not description:
+            description = "Automated snapshot by c7n of %s" % (volume)
```

Comment:
overall, looks good re pr, but curious to volume id default here given its an instrinsic attribute, is it needed? if its useful, perhaps the value would be grabbing name/title tag off the volume.. else snapshot creation date / and parent volume id feel a bit like duplicates.

Reply:
@kapilt good point. The reason we wanted to have the volume ID as part of the snapshot description is so that we can easily see it in the AWS UI. But as you pointed you, that data is there and can be toggled via the UI table options. I'm fine with having the default description be "Automated snapshot by c7n" or maybe "Automated snapshot by c7n - POLICY NAME". What do you think?

Reply:
w/ policy name sounds great

Reply:
@kapilt I've updated default description to include polic name